client/db/bolt/db.go

```diff
@@ -62,54 +66,66 @@ var (
 	botProgramsBucket      = []byte("botPrograms")
 	walletsBucket          = []byte("wallets")
 	notesBucket            = []byte("notes")
-	versionKey             = []byte("version")
-	linkedKey              = []byte("linked")
-	feeProofKey            = []byte("feecoin")
-	statusKey              = []byte("status")
-	baseKey                = []byte("base")
-	quoteKey               = []byte("quote")
-	orderKey               = []byte("order")
-	matchKey               = []byte("match")
-	orderIDKey             = []byte("orderID")
-	matchIDKey             = []byte("matchID")
-	proofKey               = []byte("proof")
-	activeKey              = []byte("active")
-	dexKey                 = []byte("dex")
-	updateTimeKey          = []byte("utime")
-	accountKey             = []byte("account")
-	balanceKey             = []byte("balance")
-	walletKey              = []byte("wallet")
-	changeKey              = []byte("change")
-	noteKey                = []byte("note")
-	stampKey               = []byte("stamp")
-	severityKey            = []byte("severity")
-	ackKey                 = []byte("ack")
-	swapFeesKey            = []byte("swapFees")
-	maxFeeRateKey          = []byte("maxFeeRate")
-	redeemMaxFeeRateKey    = []byte("redeemMaxFeeRate")
-	redemptionFeesKey      = []byte("redeemFees")
-	accelerationsKey       = []byte("accelerations")
-	typeKey                = []byte("type")
 	credentialsBucket      = []byte("credentials")
-	seedGenTimeKey         = []byte("seedGenTime")
-	encSeedKey             = []byte("encSeed")
-	encInnerKeyKey         = []byte("encInnerKey")
-	innerKeyParamsKey      = []byte("innerKeyParams")
-	outerKeyParamsKey      = []byte("outerKeyParams")
-	legacyKeyParamsKey     = []byte("keyParams")
-	epochDurKey            = []byte("epochDur")
-	fromVersionKey         = []byte("fromVersion")
-	toVersionKey           = []byte("toVersion")
-	fromSwapConfKey        = []byte("fromSwapConf")
-	toSwapConfKey          = []byte("toSwapConf")
-	optionsKey             = []byte("options")
-	redemptionReservesKey  = []byte("redemptionReservesKey")
-	refundReservesKey      = []byte("refundReservesKey")
-	byteTrue               = encode.ByteTrue
-	backupDir              = "backup"
-	disabledRateSourceKey  = []byte("disabledRateSources")
-	walletDisabledKey      = []byte("walletDisabled")
-	programKey             = []byte("program")
```

Comment:
Sorry for bloating the diff here, but we had bucket keys, value keys, and values all mixed up in this `var` block.